src/content/news/2025-05-04-maplibre-newsletter-april-2025/index.mdx

```diff
@@ -0,0 +1,138 @@
+
+- **Action Journal for Crash Debugging**: We are adding an 'action journal' to MapLibre Native. This is a **persistent log of internal events**, written out as JSON files. You can use this as a sort of black box to get information what the library was doing when rare, difficult to reproduce crashes and bugs happen. Please check out [the PR](https://github.com/maplibre/maplibre-native/pull/3409) and tell us if this is useful. We expect this to be mostly used by mission-critical enterprise type applications, which tend to have extremely low acceptable crash rates.
+
+- **Efficient Memory Management with Dynamic Texture Atlas**: Work on implementing a [dynamic texture atlas](https://github.com/maplibre/maplibre-native/pull/3198) is almost complete. Testing indicates that reduction in memory may be more significant than expected! This can also be useful when using MapLibre Native for server-side rendering. We are very interested to hear your findings on memory usage in production when releases containing this functionality are out.
+
+- **Plugin Support for Custom Layers (WIP)**: There is a [draft PR](https://github.com/maplibre/maplibre-native/pull/3430) that adds plugin functionality to MapLibre Native, allowing **custom layer types** to be added at runtime. Allowing the library to be extended more easily from the outside is something that has been talked about a long time. Although the PR in question is very much a work-in-progress, it is exciting to see some movement in this area which many users will be following with interest.
+
+- **Call for Accessibility Feedback**: We are interested in input from **accessibility** experts or developers who developed accessible maps in the past. Do you have any ideas for MapLibre Native features or perhaps style spec additions that can benefit accessibility? If yes, then please [add to the discussion](https://github.com/maplibre/maplibre-native/discussions/3439).
+
+- Releases this month:
+  - MapLibre iOS [6.12.2](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.2), [6.12.3](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.3)
+  - MapLibre Android [11.8.6](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.6)
+  - MapLibre Node.js [6.1.0](https://github.com/maplibre/maplibre-native/releases/tag/node-v6.1.0).
+
+## Web
+
+We released two versions this month: - [v5.3.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.3.1): A minor update focused on bug fixes. - [v5.4.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.4.0): A major release introducing Level of Detail (LOD) controls based on pitch and zoom levels, giving developers more flexibility in how data is rendered dynamically.
+
+Looking ahead, map-state persistence and expanded hillshade options are in the pipeline, stay tuned! There’s also early progress on a plugin to support complicated scripting languages, a feature that’s been discussed in previous community posts.
```

Comment:
What are community posts? Maybe you can add a link?

Reply:
I meant previous newsletter posts.